dashboard/app/models/school.rb

```diff
@@ -421,6 +421,30 @@ def self.seed_from_s3
           }
         end
       end
+
+      # Some of this data has #- appended to the front, so we strip that off with .to_s.slice(2) (it's always a single digit)
```

Comment:
This comment was at the start of each of the last few years' sections, but I'm not convinced it's still being used since [in the previous one-off scripts](https://github.com/code-dot-org/code-dot-org/blob/staging/bin/oneoff/nces_data/import_school_stats_by_years_2020_2021_ccd#L87) we still check for `#-value` (such as `1-Yes`).